ScreenControl_Unity/Assets/ScreenControl/Client/Scripts/Connection/ConnectionManager.cs

```diff
@@ -45,6 +45,10 @@ public static ServiceConnection serviceConnection
         // be listened to.
         public static event ClientInputActionEvent TransmitInputAction;
 
+        public delegate void HandPresenceEvent();
+        public static event HandPresenceEvent HandFound;
+        public static event HandPresenceEvent HandsLost;
```

Comment:
As these don't have params, you can remove the delegate and these can be:

'public static event Action HandFound'
'public static event Action HandsLost'

This might avoid confusion with the 'HandPresenceEvent's elsewhere